middleware.go

```diff
@@ -77,6 +78,12 @@ var DefaultProdStack = []rest.Middleware{
 }
 
 func SetupMiddleware(c config.ConfigReader, api *rest.Api) {
+
+	api.Use(&customheader.CustomHeaderMiddleware{
+		HeaderName:  "X-DEPLOYMENTS-VERSION",
+		HeaderValue: CreateVersionString(),
```

Comment:
Yes thats when variables are not set during build, on locally this can be done if someone needs to, but otherwise "Unknown" to make this clear.